### PR TITLE
document platform notifications

### PR DESCRIPTION
**Pull request by henryde**

### CODE REVIEW

File: docs/meshcloud.profile.md [hunk 1]
```diff
@@ -35,3 +35,5 @@ After a successful registration the next step is to Login into your federated Me
 ## Profile
 
 Navigate to the **Profile** Page via the navigation on the top right of the Meshpanel. Here you can find all your personal information stored in the Meshportal. You can also expand an area on the bottom of the page, that shows you all systems related to the Meshportal, that also persist your personal information. In general the Meshportal tries to store as little of your personal information as possible in the different systems.
+
+Here you can also manage your subscriptions if you wish to receive emails for any [platform notifications](administration.platforms.md).
```
Comment (sttomm) [inline docs/meshcloud.profile.md]: d: You should split up the profile page to the two section "User information" and "Platform Subscriptions". Also mention that you can subscribe to platform subscriptions per platform instance. An explanation of why the severity level can be selected and what this exactly means (e.g. that the severity info also includes warn and error) would also be helpful.

File: docs/administration.platforms.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+id: administration.platforms
+title: Platform Notifications
+---
+
+Administrators can inform users about platform specific events by creating platform notifications:
```
Comment (sttomm) [inline docs/administration.platforms.md]: d: Mention some use cases on when a platform notification might make sense. Platform outages or issues, planned maintenance windows, etc. Relating to these examples you can also explain which severity level fits best for whoch example.

File: docs/administration.platforms.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- **Platforms**: Select the platforms relevant to your notification so they can be shown on the affected platform's dashboards. Users can also [subscribe](meshcloud.profile.md#profile) to notifications by platform and will receive new notifications via email.
```
Comment (sttomm) [inline docs/administration.platforms.md]: d: you could also mention the the display of platform notifications for users on the docs page for my projects and/or project dashboard. DO we have a docs page for that atm?

Comment (henryde) [reply docs/administration.platforms.md]: We don't have docs for project overview/dashboards: MS-1818

**Pull request merged**